### ISSUE TITLE
Auto select needed group in springfox.js

### ISSUE THREAD

**Issue by alex-konkov** [springfox/springfox#1434]
It's now impossible to use links when some operation from non-default API group is is referenced. This is a feature request to add an ability to read group from e.g. query parameter as follows:
`http://localhost:8080/swagger-ui.html?group=X`

This can be done in springfox.js by selecting proper group "by default" right after all group options are fetched from the server. Then, relative links to operations (dashed) would work fine.

**Comment by dilipkrish** [comment]
PR perhaps?

**Comment by alex-konkov** [comment]
I have a hacky solution that works one way only. E.g. you put ?group=X and needed group gets autoselected. But it doesn't work the other way around, e.g. when you switch it in combo box and the URL has to be updated as well.

**Comment by dilipkrish** [comment]
May be a better idea to add default group in the `UiConfiguration` and modify springfox.js to use it.

**Comment by alex-konkov** [comment]
I don't believe this would help because it's needed to send link to any operation from any group. Right now it's simply impossible (unless it's in default group), even though URL has hash reference to the operation. The links "assume" concrete group (a default one).

Use case 1 (working):
1. Have 3 group, A (default), B
2. Copy browser URL to an operation X from A
3. Open link in new window => works, it expands to X.

Use case 2 (not working):
1. Have 3 group, A (default), B
2. Copy browser URL to an operation from B
3. Open link in new window => doesn't expand to X.

Now just imagine this happend for collaboration between server and front-end team, and having an ability to drop an openable URL with newly implemented service is a very useful feature. But it's limited only to one group, which if unfortunate. This is not about changing the default group, as you can see.

**Comment by dilipkrish** [comment]
@alex-konkov I think given a multi group setup. Its always going to be a challenge, because swagger-ui is not geared to handle that. The light shim we provide in springfox.js can only do so much

**Comment by TomJahncke** [comment]
Is there way to have a default group so when navigating to swagger-ui.html?
